extensions/amp-story-dev-tools/0.1/amp-story-dev-tools-tab-preview.css

```diff
@@ -0,0 +1,269 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+ amp-story-dev-tools-tab-preview {
+  --i-amphtml-story-dev-tools-device-shadow: 0px 11.0046px 27.5114px rgba(0, 0, 0, 0.34), inset 0px 2.20091px 3.30137px rgba(44, 46, 52, 0.6) !important;
+
+  --i-amphtml-story-dev-tools-device-gray-1: #101010 !important;
+  --i-amphtml-story-dev-tools-device-gray-2: #2E3137 !important;
+  --i-amphtml-story-dev-tools-device-gray-3: #2E2E2E !important;
+  --i-amphtml-story-dev-tools-device-gray-4: #3D3D3D !important;
+  --i-amphtml-story-dev-tools-device-gray-transparent-1: #111111CC !important;
+  --i-amphtml-story-dev-tools-device-white-transparent-1: #FFFFFF22 !important;
+  --i-amphtml-story-dev-tools-device-white-transparent-2: #FFFFFF77 !important;
+  --i-amphtml-story-dev-tools-device-white-transparent-3: #FFFFFFCC !important;
+
+  --i-amphtml-story-dev-tools-material-easing: cubic-bezier(0.4, 0.0, 0.2, 1);
```

Comment:
`-accelerated`

Should these live in the `amp-story-dev-tools.css` file?

Reply:
We don't use the animations anywhere else TBH, since all the other tabs are static. We can move them to the main CSS file if we want to use them somewhere else, but keep them here until then.

Reply:
What's the likelihood that we forget about it and declare new ones as we're doing in this file? 🤷

Reply:
Sure! I'm also moving the colors to the main CSS